apps/server/src/modules/files/uc/delete-files.uc.ts

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable no-await-in-loop */
 import { DeleteObjectCommand, S3Client } from '@aws-sdk/client-s3';
 import { Injectable } from '@nestjs/common';
-import { File, StorageProvider } from '@shared/domain';
-import { FilesRepo } from '@shared/repo';
+
+import { StorageProviderEntity } from '@shared/domain';
 import { StorageProviderRepo } from '@shared/repo/storageprovider/storageprovider.repo';
 import { LegacyLogger } from '@src/core/logger/legacy-logger.service';
```

Comment:
```suggestion
import { LegacyLogger } from '@src/core/logger';
```

Reply:
Done, I didn't add it myself but thanks for spotting it, I've fixed it.